### PR TITLE
Allow Node 16

### PR DESCRIPTION
**Pull request by snewcomer**
https://github.com/emberjs/data/pull/7608/checks?check_run_id=2959002812

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -70,7 +70,7 @@
     "qunit-dom": "^1.6.0"
   },
   "engines": {
-    "node": "12.* || 14.*"
+    "node": "12.* || >= 14"
```
Comment (btecu) [inline package.json:73]: Maybe `12.* || 14.* || >= 16`?
Node.js 15 is no longer supported.

Comment (snewcomer) [reply package.json]: great idea!

**Comment by btecu** [comment]
Thanks @snewcomer!

**Comment by snewcomer** [comment]
Thanks!  Lmk when you get a new release and then we can get our ember-data builds back up and running!

**Comment by btecu** [comment]
Released in [0.8.3](https://github.com/btecu/ember-simple-tree/releases/tag/0.8.3).

**Pull request merged**